hypershift-operator/controllers/nodepool/secret_janitor_test.go

```diff
@@ -152,7 +151,7 @@ spec:
 	r := secretJanitor{
 		NodePoolReconciler: &NodePoolReconciler{
 			Client:          c,
-			ReleaseProvider: &fakereleaseprovider.FakeReleaseProvider{Version: supportedversion.LatestSupportedVersion.String()},
+			ReleaseProvider: &fakereleaseprovider.FakeReleaseProvider{Version: semver.MustParse("4.18.0").String()},
```

Comment:
I would add a comment that this needs to stay at this version and add some of Alberto's reasoning why from here - https://redhat-internal.slack.com/archives/G01QS0P2F6W/p1732552629847439?thread_ts=1732541618.147769&cid=G01QS0P2F6W